modules/govuk_ci/manifests/agent/docker.pp

```diff
@@ -10,4 +10,13 @@
 
   include ::docker::compose
 
+  # Docker can hog a lot of space by not cleaning up after itself, this removes
+  # dangling images and volumes.
+  $remove_dangling_images = 'docker rmi $(docker images --filter "dangling=true" -q --no-trunc)'
+  $remove_dangling_volumes = 'docker volume rm $(docker volume ls -qf dangling=true)'
+  file { '/etc/cron.d/clean_dangling_docker_files':
```

Comment:
Rather than using a raw file could this be done with the crondotdee resource which will give you a more readable form? ([for example](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/node/s_mirrorer.pp#L20) & [source](https://github.com/alphagov/govuk-puppet/blob/master/modules/cron/manifests/crondotdee.pp))